Makefile

```diff
@@ -4,8 +4,19 @@ unit:
 
 INTEGRATIONS = "@abi or @algod or @applications or @applications.verified or @assets or @auction or @c2c or @compile or @dryrun or @dryrun.testing or @indexer or @indexer.231 or @indexer.applications or @kmd or @rekey_v1 or @send.keyregtxn or @send or @compile.sourcemap"
 integration:
-	behave --tags=$(INTEGRATIONS) tests -f progress2
+	behave --tags=$(INTEGRATIONS) tests -f progress2 --no-capture
+
+# channel or source
+TYPE ?= channel
```

Comment:
I think this should be removed here and only in the `.env` so there is one central place to control these vars

Reply:
After playing around some more here, I'm convinced. The less places to overwrite, the easier it is to get it right and use.

Reply:
I sent the change up.